internal/infrastructure/db/migrations/20231113225121_create_template_dd.sql

```diff
@@ -0,0 +1,29 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+CREATE TABLE IF NOT EXISTS template_device_definitions (
+    id BIGSERIAL,
+    device_definition_id BIGINT NOT NULL,
```

Comment:
device_definition_id is a KSUID in the device definitions database, so this one we do want to change to a char(27)
```suggestion
    device_definition_id char(27) NOT NULL,
```